# Request
bool attach_to_shelf
---
# Response
bool complete